lib/galaxy/model/__init__.py

```diff
@@ -5980,6 +5980,11 @@ def to_dict(self, view="collection"):
 
 
 class LibraryDatasetDatasetAssociation(DatasetInstance, HasName, Serializable):
+    message: Mapped[Optional[str]] = mapped_column(TrimmedString(255))
```

Comment:
@nsoranzo the `message` and `tags` attribute are already declared via imperative mapping at the bottom of the module. This syntax is for declarative mapping and shouldn't be used in this class (it's not mapped declaratively, so both statements are ignored). Sorry I missed this! Can we remove them, or was there a reason for adding them?

Reply:
@jdavcs The reason is type annotation, the `message` and `tags` attributes of objects of this class were not declared and it was already an issue before: https://github.com/galaxyproject/galaxy/pull/18773/files#diff-90e35f587e5b32cb4c2f34d143189d062558cd90c2ccb3ed5a891cc735dc1a24L508

I can remove the assignments (` = mapped_column(...)` and ` = relationship(...)` respectively) and leave only the type annotation if that works for you?

Reply:
Yes, please. That would remove restating the relationship.